### PR TITLE
Server config choice

### PR DESCRIPTION
**Pull request by RyanIkhlef**
##### TODOs

- [x] Check iOS support
  - [x] Flash cannot be enabled 
- [x] Increase minor version
- [x] Update release notes

### CODE REVIEW

**Comment by Alystrasz** [comment]
Regarding newest settings menu section design, do not hesitate taking all available screen space.

1. I think that we should have an entire subsection dedicated to backend configuration, so let's use separators.
2. Be careful with buttons, both should have same style and properly display inner text.
3. Since there's a lot of settings in this menu, maybe we should start adding subtitles to subsections?

--- 

<p align="center">

|  |  |
|:--:|:--:|
| *App screenshot* | *Quick mockup made on GIMP* |

</p>

File: apolline-flutter/lib/models/server_endpoint_handler.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  setDefaultConfig() async {
+    ServerModel mainConfig = ServerModel(
+        GlobalConfiguration().get(ApollineConf.API_URL),
+        GlobalConfiguration().get(ApollineConf.PING_URL),
+        GlobalConfiguration().get(ApollineConf.USERNAME),
+        GlobalConfiguration().get(ApollineConf.PASSWORD),
+        GlobalConfiguration().get(ApollineConf.DBNAME)
+    ); // Récupération de la config du fichier config_dev.json
+
+    ServerModel defaultConfig = await SqfLiteService().getDefaultEndpoint(); // Si une configuration par defaut à déjà été enregistré, on la met sinon on met la config du fichier config_dev.json
```
Comment (Alystrasz) [inline apolline-flutter/lib/models/server_endpoint_handler.dart:27]: Please write all code comments in English.

**Review by Alystrasz — CHANGES_REQUESTED** [review]
There are some changes to translations that must be done, but great job overall!

File: apolline-flutter/assets/translations/en-GB.json [hunk 2]
```diff
@@ -112,5 +118,29 @@
...
+      "username": "Username :",
+      "password": "Password :",
+      "dbname": "Database name : ",
+      "confirm": "Save"
+    }
+  },
+  "endpointQRScanner": {
+    "title": "Scan QR code",
+    "permissionRequired": "Access to the camera permission is required.",
+    "codeTypeError": "You must to scan a QR code.",
```
[the remark this answers is not in the document]
Comment (RyanIkhlef) [reply apolline-flutter/assets/translations/en-GB.json:137]: It's shown when user scan an other type of code (bar code for example).

**Review by Alystrasz — APPROVED** [review]

**Comment by Alystrasz** [comment]
Looks good to me.
You can increase minor version in `pubspec.yaml`, and if it works on iOS, I'll merge.

**Pull request merged**